src/scripts/main.js

```diff
@@ -1,7 +1,95 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+// Ініціалізація гри
+import { WIN, LOSE, Game } from '../modules/Game.class';
 
-// Write your code here
+const game = new Game(null, showWinMessage, showLoseMessage);
+
+const startButton = document.querySelector('.start');
+const cells = document.querySelectorAll('.field-cell');
+const score = document.querySelector('.game-score');
+const recordElement = document.querySelector('.best__score');
+const messageContainer = document.querySelector('.message-container');
+
+function showWinMessage() {
+  messageContainer.classList.remove('hidden');
+  messageContainer.classList.add('message-win');
+  messageContainer.textContent = 'You Win!';
+}
+
+function showLoseMessage() {
+  messageContainer.classList.remove('hidden');
+  messageContainer.classList.add('message-lose');
+  messageContainer.textContent = 'You Lose!';
+}
+
+function hideMessage() {
+  messageContainer.classList.add('hidden');
+}
+
+function updateScore(value) {
+  score.textContent = value;
+}
+
+function saveScore(sscore) {
+  const currentRecord = localStorage.getItem('highScore');
+
+  if (!currentRecord || sscore > currentRecord) {
+    localStorage.setItem('highScore', sscore);
+    displayRecord(); // Оновлюємо рекорд на екрані
+  }
+}
+
+// Функція для відображення рекорду
+function displayRecord() {
+  const highScore = localStorage.getItem('highScore') || 0;
+
+  recordElement.textContent = highScore;
+}
+
+displayRecord();
+
+// Оновлення рекорду, коли він збільшується
+game.getScore = function (moved, value) {
+  if (moved && value > 0) {
+    this.score += value; // Додаємо до поточного рахунку
+    updateScore(this.score); // Оновлюємо відображення рахунку
+    saveScore(this.score); // Перевіряємо, чи це новий рекорд, і зберігаємо його
+  }
+};
+
+// Обробка натискання на кнопку старту
+startButton.addEventListener('click', () => {
+  startButton.textContent = 'Restart';
+  startButton.classList.remove('start');
+  startButton.classList.add('restart');
+
+  cells.forEach((cell) => {
+    cell.classList.remove('hidden');
+  });
+
+  if (game.status === WIN || game.status === LOSE) {
+    hideMessage();
```

Comment:
You should remove start message:

![image](https://github.com/user-attachments/assets/4ed50ffb-f213-4cff-9e12-6ba7b8eadb72)

Reply:
Also, add margin to Lose message
![image](https://github.com/user-attachments/assets/00f531b6-dc49-40d0-aeef-f227d935e2f8)